src/main/java/com/GotYourBac/gotYourBac/controllers/HomeController.java

```diff
@@ -0,0 +1,24 @@
+package com.GotYourBac.gotYourBac.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class HomeController {
+
+    //Apparently GetMapping without the route will still give us what we want.
+    @GetMapping("/")
+    public String gethome(){
+        return "home";
+    }
+
+    @GetMapping("/balmerpeekinfo")
+    public String getBalmerPeekInfo() {
+        return "balmerpeek";
+    }
+
+    @GetMapping("/error")
```

Comment:
we didn't cover custom error pages, but since you are tackling it, this article will help you make them actually describe the error thrown
https://www.baeldung.com/spring-boot-custom-error-page

Reply:
Hollie will take this feature and create a new branch to knock this out